packages/electrode-ignite/lib/menu-items/electrode-oss-doc.js

```diff
@@ -5,9 +5,10 @@ const Promise = require("bluebird");
 const opn = require("opn");
 const chalk = require("chalk");
 
-module.exports = function() {
+module.exports = function(link) {
```

Comment:
should add a `docs` so the `logger.log` below doesn't always log the same `"Electrode docs"`.

Reply:
done @jchip